README.md

```diff
@@ -13,33 +13,74 @@ Native execution requires the following:
 - [Nodejs v14](https://nodejs.org/en/) 
 - [yarn](https://yarnpkg.com/)
 - [RW API](https://api.resourcewatch.org/)
-- [Redis](https://redis.io/)
+- [Redis](https://redis.io/) (optional)
 
 There are included [Dockerfile](https://docs.docker.com/engine/reference/builder/) and [docker compose](https://docs.docker.com/compose/) configuration files that may make it easier to run the application locally.
 
 # Installation
-We strongly recommend to use [NVM](https://github.com/nvm-sh/nvm) to handle different Node versions.
+
+Be sure you are using the correct Nodejs version. We recommend using [NVM](https://github.com/nvm-sh/nvm) to handle different Node versions.
+
+Begin by installing the necessary nodejs dependencies, using `yarn`:
```

Comment:
Not all dependencies are node-related, I would rename it to just `dependencies` or similar.